generators/server/prompts.js

```diff
@@ -182,14 +182,6 @@ function askForServerSideOpts(meta) {
                             name: 'No database'
                         });
                     }
-                    if (
-                        (response.authenticationType !== 'oauth2' && applicationType === 'microservice')
-                        || (response.authenticationType === 'uaa' && applicationType === 'gateway')
-                    ) {
-                        opts.push({
-                            value: 'no',
-                            name: 'No database'
-                        });
                     }
```

Comment:
I think you need to include this additional `}` in the deleted part too

Reply:
sure!

Reply:
sorry, I wasn't clear enough: you need to delete `}`, not add a new one. You can check your code with npm test or yarn test

Reply:
right, travis will test your last commit